test/checkpoint/test_checkpoint.c

```diff
@@ -130,8 +130,7 @@ main(int argc, char *argv[])
 	/* Clean up on signal. */
 	(void)signal(SIGINT, onint);
 
-	if ((ret = testutil_work_dir_from_path(g.home, 512, working_dir)) != 0)
-		testutil_die(ret, "provided directory name is too long");
+	(void)testutil_work_dir_from_path(g.home, 512, working_dir);
```

Comment:
@daveh86 There is no need to cast the result of functions that return void to void. That's only necessary when not using the return value from functions that return non-void.